src/mjolnir/ingest_transit.cc

```diff
@@ -457,58 +517,45 @@ bool write_stop_pair(Transit& tile,
         uniques.lock.unlock();
       }
 
-      bool origin_is_generated = tile_info.station_children.find({origin_stopId, currFeedPath}) !=
-                                 tile_info.station_children.end();
-      auto origin_onestop_id =
-          origin_is_generated ? origin_stopId + "_" + to_string(NodeType::kMultiUseTransitPlatform)
-                              : origin_stopId;
       stop_pair->set_origin_onestop_id(origin_onestop_id);
-
-      bool dest_is_generated = tile_info.station_children.find({dest_stopId, currFeedPath}) !=
-                               tile_info.station_children.end();
-      auto dest_onestop_id = dest_is_generated
-                                 ? dest_stopId + "_" + to_string(NodeType::kMultiUseTransitPlatform)
-                                 : dest_stopId;
       stop_pair->set_destination_onestop_id(dest_onestop_id);
 
+      stop_pair->set_destination_arrival_time(dest_stopTime.arrival_time.get_total_seconds());
+      stop_pair->set_origin_departure_time(origin_stopTime.departure_time.get_total_seconds());
```

Comment:
this was also a bug: we have to set these attributes regardless if the locations are inside the tile or not. otherwise they wouldn't be set at all for some pairs which breaks lots of stuff